purchases/src/main/kotlin/com/revenuecat/purchases/paywalls/PaywallData.kt

```diff
@@ -35,6 +36,30 @@ data class PaywallData(
     @SerialName("localized_strings") internal val localization: Map<String, LocalizedConfiguration>,
 ) {
 
+    /**
+     * Returns the [LocalizedConfiguration] to be used based on the current locale.
+     */
+    val localizedConfiguration: LocalizedConfiguration
+        get() {
+            val preferredLocales = LocaleListCompat.getDefault()
+
+            val localizedConfigurations = mutableListOf<LocalizedConfiguration>()
+            for (i in 0 until preferredLocales.size()) {
+                preferredLocales.get(i)?.let { locale ->
+                    configForLocale(locale)?.let { localizedConfiguration ->
+                        localizedConfigurations.add(localizedConfiguration)
```

Comment:
Hmm couldn't we just return the moment we find one compatible?

Reply:
Yeah 👍🏻  iOS uses `first(where:)` for that.